Assets/Cartoon_City_Mega-Pack/Park_pack/skyboxes/Textures/DawnDusk Skybox.mat

```diff
@@ -47,7 +47,9 @@ Material:
         m_Texture: {fileID: 2800000, guid: a6de6086ecd5fe84699b0bc7e00608a4, type: 3}
         m_Scale: {x: 1, y: 1}
         m_Offset: {x: 0, y: 0}
-    m_Floats: []
```

Comment:
The code Mikkel wrote changes the material instead of it's instance for some reason. Should we look into this?

Reply:
Yeah maybe a bit later this should be looked into